spec/rest/auth.spec.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+define(['ably', 'shared_helper'], function(Ably, helper) {
+  beforeAll(helper.setupApp);
+  afterAll(helper.tearDownApp);
+
+  describe('REST Auth', function() {
+    var currentTime; // set by first test
+
+    describe('Ably.Rest#time', function() {
+      it('obtains the server time', function(done) {
+        helper.AblyRest({ key: 0 }).time(function(err, time) {
+          if(err) { throw err; }
```

Comment:
Would it be possible to replace this with an expect statement (`expect(err).toBeEmpty()` or something)? I suppose Jasmine will fail anyway but I think it would be more descriptive with an expect statement.